src/commands/services/export.ts

```diff
@@ -119,12 +120,14 @@ function getServiceMapper(
         i: service.serviceId,
         n: service.serviceName,
         d: service.serviceMetadata?.description,
-        sc: service.serviceMetadata?.scope || ServiceScopeEnum.NATIONAL
+        sc: service.serviceMetadata?.scope || ServiceScopeEnum.NATIONAL,
+        q: ValidService.is(service) // ? 1 : 0
       };
     }
     return {
       i: service.serviceId,
-      n: service.serviceName
+      n: service.serviceName,
+      q: ValidService.is(service) // ? 1 : 0
```

Comment:
@BurnedMarshal maybe I made a mistake but `ValidService.is(service)` result is everytime false

Reply:
@pasqualedevita fixed in edb8c53
we need to use `decode` instead of `is` because the service come from a raw query and not from the io-model.
Only the `decode` method support array in `authorizedCIDRs` and `authorizedRecipients` sets.